app/src/main/java/org/apache/fineract/ui/online/groups/grouptasks/GroupTasksBottomSheetFragment.kt

```diff
@@ -120,6 +120,11 @@ class GroupTasksBottomSheetFragment(val group: Group) : FineractBaseBottomSheetD
 
     @OnClick(R.id.btn_submit_task)
     fun submitTask() {
+        if (rootView.et_comment.text.toString().isEmpty()) {
```

Comment:
Looks Good to me. With kotlin you can directly access UI elements
```suggestion
        if (et_comment.text.toString().isEmpty()) {
```

Reply:
> Looks Good to me. With kotlin you can directly access UI elements

I already did that but it was throwing null pointer, that's why the rootview is there actually the rootview is consisting the rootview and it is this class is actually handling the rootview so that's why rootview is used to access et_comment.